### ISSUE TITLE
RangeError: "port" argument must be >= 0 and < 65536

### ISSUE THREAD

**Issue by JacksonGariety** [FWeinb/electron-screenshot-service#19]
Running the code from the README produces this error with the following stacktrace:

```
RangeError: "port" argument must be >= 0 and < 65536
    at assertPort (internal/net.js:17:11)
    at Server.listen (net.js:1382:5)
    at ReqSocket.Socket.bind (/Users/Jackson/Sites/topsters2/node_modules/axon/lib/sockets/sock.js:398:15)
    at /Users/Jackson/Sites/topsters2/node_modules/electron-screenshot-service/src/browser.js:18:8
    at new Promise (/Users/Jackson/Sites/topsters2/node_modules/babel-polyfill/node_modules/core-js/modules/es6.promise.js:193:7)
    at bindSocket (/Users/Jackson/Sites/topsters2/node_modules/electron-screenshot-service/src/browser.js:17:22)
    at Object.module.exports.createBrowser (/Users/Jackson/Sites/topsters2/node_modules/electron-screenshot-service/src/browser.js:77:10)
    at Object.module.exports.screenshot (/Users/Jackson/Sites/topsters2/node_modules/electron-screenshot-service/src/browser.js:67:16)
    at _callee8$ (/source/server.js:182:19)
    at tryCatch (/Users/Jackson/Sites/topsters2/node_modules/babel-regenerator-runtime/runtime.js:61:40)
```

**Comment by FWeinb** [comment]
Could you please add the node version and OS your are using. Can't reproduce it on OS X/Linux.

**Comment by JacksonGariety** [comment]
@FWeinb OS X 10.11 and node `v6.0.0`

**Comment by FWeinb** [comment]
Published in `3.1.1`. Thanks for reporting 👍
